tests/integration/backend/reqif/profiles/p01_sdoc/end_to_end/--multiline-is-xhtml/01_option_is_provided/test.itest

```diff
@@ -0,0 +1,14 @@
+RUN: %strictdoc export --formats=reqif-sdoc --multiline-is-xhtml %S/sample.sdoc
```

Comment:
Implement and write another test where there is `multiline_is_xhtml` in strictdoc.toml.